src/explore-education-statistics-admin/src/modules/find-statistics/components/ReleaseContentAccordion.tsx

```diff
@@ -49,26 +44,20 @@ const ReleaseContentAccordion = ({
         >
           {content.map(
             ({ id, heading, caption, order, content: contentdata }, index) => (
-              <EditingContentBlockContext.Provider
+              <AccordionSection
+                id={id}
+                index={index}
+                heading={heading || ''}
+                caption={caption}
                 key={order}
-                value={{
-                  ...releaseContext,
-                  sectionId: id,
-                }}
               >
-                <AccordionSection
-                  id={id}
-                  index={index}
-                  heading={heading || ''}
-                  caption={caption}
-                >
-                  <ContentBlock
-                    content={contentdata}
-                    id={`content_${order}`}
-                    publication={release.publication}
-                  />
-                </AccordionSection>
-              </EditingContentBlockContext.Provider>
+                <ContentBlock
+                  sectionId={id}
+                  content={contentdata}
+                  id={`content_${order}`}
+                  publication={release.publication}
+                />
+              </AccordionSection>
```

Comment:
Isn't it expected that there will be multiple contentblocks in a section? - here we suggest that there will be only one static `contentData` block

Reply:
on second look, I see that `content: contentData` is supposed to be an array of blocks.
So all is fine